temp/OnnxTransformTempSolution/OnnxModel/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "dependencies": [],
+  "description": "",
+  "location": "",
+  "name": "fc1",
+  "params": {
+    "model": {
+      "frameworkVersion": "",
+      "graphVersion": 0,
+      "interfaces": [
+        {
+          "inputs": [
+            {
+              "description": "",
+              "internalName": "Input3",
+              "name": "input0",
```

Comment:
@Jignesh - manifest.json file changes the "internalName" of nodes to "name". That's something to take care of when removing the dependency on manifest file.